src/wasm/response.rs

```diff
@@ -117,6 +121,25 @@ impl Response {
         buffer.copy_to(&mut bytes);
         Ok(bytes.into())
     }
+   
+    /// Convert the response into a `Stream` of `Bytes` from the body.
+    #[cfg(feature = "stream")]
+    pub fn bytes_stream<'a>(self) -> LocalBoxStream<'a, crate::Result<Bytes>> {
```

Comment:
Should this return `impl Stream<....>` to match `bytes_stream` in `async_response`? And so it's not a breaking change if the `futures_util` crate gets updated.

Reply:
Yeah, that makes sense. I can't edit or take over this merge request so I created a new one: https://github.com/seanmonstar/reqwest/pull/1713